content/en/docs/concepts/storage/volumes.md

```diff
@@ -858,7 +858,7 @@ operations from the existing in-tree plugin to the `rbd.csi.ceph.com` {{<
 glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this
 feature, the
 [Ceph CSI driver](https://github.com/ceph/ceph-csi)
-must be installed on the cluster and the `CSIMigration` and `CSIMigrationRBD`
+must be installed on the cluster and the `CSIMigration` and `csiMigrationRBD`
```

Comment:
Per the [docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/), the current format is accurate and appropriate  since while discussing API objects UpperCamelCase is used.